client/state/reducer.js

```diff
@@ -23,6 +23,7 @@ import atomicHosting from './hosting/reducer';
 import atomicTransfer from './atomic-transfer/reducer';
 import billingTransactions from './billing-transactions/reducer';
 import checklist from './checklist/reducer';
+import concierge from './concierge/reducer';
```

Comment:
These changes were needed for the Concierge query component to populate state with the API response (without it, the `getConciergeNextAppointment` selector was always `null`). Which makes sense – I've just never seen a set of reducers _not_ be in the root reducer, and I'm surprised that, like, all of Concierge isn't broken without it. Anyone see what's going on (or maybe my limited Redux knowledge has me not seeing something)? ¯\_(ツ)_/¯

Reply:
Which routes? It could be that it didn't work before

Reply:
Ah found it https://github.com/Automattic/wp-calypso/pull/30480 so we probably don't want to re-add here

Reply:
Ah, I guess this is related; so the Concierge reducers are only being registered for select routes, and I've gone and shoved them in for all routes?.. This shows up when proceeding to a concierge route in this PR:

<img width="561" alt="Screen Shot 2020-04-14 at 4 45 29 PM" src="https://user-images.githubusercontent.com/349751/79284482-a13fbd00-7e6f-11ea-8c7f-9f5bb39448bb.png">

Reply:
Yeah I think we were trying to slim down the main reducer bundle, it'd be worth chatting with @Automattic/team-calypso to see if it makes sense to add back. (As we need this in the MySites default page).

Reply:
@gwwar ah, didn't see your comment, thanks! That's interesting. So we would do the same async load thing as that PR, but only if the given site has a plan qualifying for Quick Start sessions?

Reply:
> but only if the given site has a plan qualifying for Quick Start sessions?

Yeah, let's give it a try. If we open this card up to everyone on a free plan, it probably would make more sense to add back to the main reducer since this is the default section we land on.

Reply:
I got this working in 5aea6d8437, but it feels really heavy-handed and... special compared to always having it in the root reducer like so much else. I'm not sure how to measure the performance impact though, done this way. @gwwar @jsnajdr any preferences either way?

Reply:
@jsnajdr's on leave. Overall, @Automattic/team-calypso @griffbrad I think we'll eventually want to add this back to the main bundle when the card is visible to folks on all plans.

For now I could go either way on adding it back for simplicity or trying to keep another async load. @kwight for the perf impact I suppose we could profile locally trying async vs the reducer bundle with caches disabled.

Reply:
I'll eave it async for now, and we can worry about moving to the root reducer if/when we open the card to all users.